sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinModelsRestClient.cs

```diff
@@ -21,37 +21,27 @@ internal partial class DigitalTwinModelsRestClient
             CreateModelsOptions digitalTwinModelsAddOptions = null,
             CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope("DigitalTwinModelsClient.Add");
```

Comment:
here and below - all of these protocol layer calls have corresponding convenience layer calls in which the diagnostic scope is created and started.